src/shared/components/upgradeBrowser/upgradeBrowser.css

```diff
@@ -0,0 +1,63 @@
+.layout {
+  position: absolute;
+  top: 25%;
+  left: 20%;
+  border: 6px solid grey;
+  background: #f0f2f2;
+  overflow: auto;
+  border-radius: 4px;
+  outline: none;
+  padding: 20px;
+  width: 59%;
+  height: 43%;
+  font-size: 2em;
+}
+
+.button {
+  position: absolute;
+  bottom: 10%;
+  right: 5%;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: white;
+}
+
+.warning_logo {
+  color: #d1665a;
+  font-size: 2em;
+}
+
+.content_align {
+  text-align: center;
+}
+
+.upgrade_browser_message {
+  font-weight: bold;
+}
+
+.main_message {
+  font-size: 20px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.flex {
+  display: flex;
```

Comment:
Please add `justify-content: center`, so the list of browsers is centered on the flex-axis (horizontally).